packer/instance-scripts/init-quorum.sh

```diff
@@ -274,6 +274,22 @@ function wait_for_terraform_provisioners {
     done
 }
 
+# If using EIPs, associate with instance
```

Comment:
Super nit but could you throw this in a function?  I think it's complex and specific enough to put it in a function.

I'd pass the `AWS_REGION` in as an argument and read `USING_EIP` from file within the function. Should be plenty of examples of both.